examples/elasticache/v1beta1/serverlesscache.yaml

```
@@ -0,0 +1,98 @@
+# SPDX-FileCopyrightText: 2024 The Crossplane Authors <https://crossplane.io>
+#
+# SPDX-License-Identifier: CC0-1.0
+
+apiVersion: elasticache.aws.upbound.io/v1beta1
+kind: ServerlessCache
+metadata:
+  annotations:
+    meta.upbound.io/example-id: elasticache/v1beta1/serverlesscache
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
```

There is a cache resource named example in our account and uptest is failing because of this, please try the following.

```suggestion
  name: example-${Rand.RFC1123Subdomain}
```